src/py/mat3ra/made/tools/utils/__init__.py

```diff
@@ -64,6 +65,23 @@ def get_distance_between_coordinates(coordinate1: List[float], coordinate2: List
     return float(np.linalg.norm(np.array(coordinate1) - np.array(coordinate2)))
 
 
+def calculate_norm_of_distances_between_coordinates(coords1: np.ndarray, coords2: np.ndarray) -> float:
+    """
+    Calculate the norm of distances between two sets of coordinates.
+
+    Args:
+        coords1 (np.ndarray): The first set of coordinates.
+        coords2 (np.ndarray): The second set of coordinates.
+
+    Returns:
+        float: The calculated norm.
+    """
+    tree = cKDTree(coords2)
+    distances, _ = tree.query(coords1)
```

Comment:
query doesn't seem to work here, should be just a simple `map(np.linalg.norm)` for the second coords, or consider https://docs.scipy.org/doc/scipy-1.13.1/reference/generated/scipy.spatial.cKDTree.sparse_distance_matrix.html#scipy.spatial.cKDTree.sparse_distance_matrix

Reply:
ok